### PR TITLE
adding php beta sdk release notes

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

Updating  FF Release Notes with PHP SDK Beta 0.2.0 
* Jira/GitHub Issue numbers (if any): \FFM-11165

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes: Version released to at least one prod environment.

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -50,6 +50,13 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
  -- increasing the capacity of dictionary, and
  -- additional cache recovery checks. (FFM-11089)
 
+### PHP SDK (Beta)
+
+#### Version 0.2.0 
+
+ - We've fixed the JWT fields missing in cache on authentication. (FFM-11123)
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:57]:
```suggestion
 - We've fixed the JWT fields missing in the cache on authentication. (FFM-11123, ZD-60936)
```

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -50,6 +50,13 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+ - We've added SDK type, version and lang to HTTP header. (FFM-7037)
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:58]:
```suggestion
 - We've added SDK type, version, and lang to the HTTP header. (FFM-7037)
```

File: release-notes/feature-flags.md (hunk 1 above)
Comment (brian-f-harness) [reply release-notes/feature-flags.md:57]: We need to add Zendesk tickets for reference when applicable. This references help customers identify fixed issues that they're tracking.

Might also consider updating to remove "We've":
Fixed the JWT fields missing in the cache on authentication. (FFM-11123, ZD-60936)

File: release-notes/feature-flags.md (hunk 2 above)
Comment (brian-f-harness) [reply release-notes/feature-flags.md:58]: Might also consider updating to remove "We've":
Added SDK type, version, and lang to the HTTP header. (FFM-7037)

**Review by brian-f-harness — APPROVED** [review]
Looks good. Please add the Zendesk ticket reference. Also added a couple optional suggestions.

File: release-notes/feature-flags.md (hunk 1 above)
Comment (jenniferopal) [reply release-notes/feature-flags.md:57]: I wasn't aware of the ZD tickets needed to be added. I'll be mindful of that for next time. Thank you.

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**